unpackaged/config/rd2_post_config/objects/npe03__Recurring_Donation__c.object

```diff
@@ -58,6 +58,30 @@
         <trackTrending>false</trackTrending>
         <type>Date</type>
     </fields>
+    <fields>
```

Comment:
I'm pretty sure that you do not need to include these fields in the object unpackaged metadata. It's worth testing, but there are other fields granted permission in the profiles (EndDate, CurrentYearValue, NextYearValue, etc.) that are not included in the object file.